src/bundle/Resources/public/js/scripts/admin.content.draft.edit.js

```diff
@@ -0,0 +1,25 @@
+(function (global, doc) {
+    const editVersion = (event) => {
+        const contentId = event.currentTarget.dataset.contentId;
+        const versionNo = event.currentTarget.dataset.versionNo;
+        const languageCode = event.currentTarget.dataset.languageCode;
+        const contentDraftEditUrl = event.currentTarget.dataset.href;
+        const checkLink = window.Routing.generate('ezplatform.version.check', { contentId, versionNo, languageCode });
+
+        event.preventDefault();
+
+        let request = $.get(checkLink);
+
+        request.fail(function(xhr) {
+            if(xhr.status === 409){
+                $('.modal-content').html(xhr.responseText);
+                $('#draft-has-conflict-modal').modal('show');
+            }
+            if(xhr.status === 301){
+                window.location.href = contentDraftEditUrl;
```

Comment:
As @webhdx  pointed this should be on success. And second thing for me this endpoint should return an `url` on success where redirect user.